examples/sas_logical_interconnect_groups.py

```diff
@@ -101,6 +101,16 @@
 sas_lig_found_by_name = oneview_client.sas_logical_interconnect_groups.get_by('name', data['name'])
 print("\n   '{name}' at uri: {uri}".format(**sas_lig_found_by_name[0]))
 
+# Get SAS Interconnect Group by scope_uris
+if oneview_client.api_version == 600:
```

Comment:
Can be `oneview_client.api_version >=600` because api 800 also has this parameter. So we would not need to change it when we add support for api800.